src/idlechecker/idle-checker.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# idle-checker.sh checks for cpu utilization of the VM and update last active timestamp if the CPU utilization
+# is above a given threshold.
+#
+# Note that this script is dependent on some variables and packages being set up:
+#
+# - AWS CLI is installed
+# - CLOUD: cloud platform VM is running on
+# - bc https://manpages.ubuntu.com/manpages/trusty/en/man1/bc.1.html
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# threshold - threshold for cpu usage, used to determine if instance is idle. If usage goes above this number count resets to zero. By default 0.1 (10 percent)
+readonly threshold="${THRESHOLD:-0.1}"
```

Comment:
You don't *need* to change the name. If you really want to, then make it THRESHOLD_VAL or some such, but this is fine:

```suggestion
readonly THRESHOLD="${THRESHOLD:-0.1}"
```